package/MDAnalysis/analysis/contacts.py

```diff
@@ -342,7 +344,7 @@ def run(self, store=True, force=False):
 
                 if store:
                     records.append((frame, q1, q2, n1, n2))
-                outbz2.write("{frame:4d}  {q1:8.6f} {q2:8.6f}  {n1:5d} {n2:5d}\n".format(**vars()))
+                outbz2.write("%(frame)4d  %(q1)8.6f %(q2)8.6f  %(n1)5d %(n2)5d\n" % vars())
```

Comment:
This syntax is discouraged in modern python. Please keep using the `format` function.